src/mocks/services/authServiceHandler.ts

```
@@ -122,18 +198,24 @@ const authServiceHandler = [
   http.post(`${BASE_URL}/user/verify/send`, async ({ request }) => {
     const { email } = (await request.json()) as RequestEmailCode;
 
-    if (email !== USER_INFO_DUMMY.email)
+    if (!EMAIL_REGEX.test(email))
       return HttpResponse.json({ message: '이메일을 다시 확인해 주세요.' }, { status: 400 });
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)